### PR TITLE
add doc for tiflash resource control

### PR DESCRIPTION
**Pull request by guo-shaoge**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

- Remove the system variable `tidb_enable_tiflash_pipeline_model`
- Control whether to enable TiFlash resource control and Pipeline Model using the TiFlash configuration item `enable_resource_control`

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: releases/release-7.2.0.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ summary: 了解 TiDB 7.2.0 版本的新功能、兼容性变更、改进提升
 
 TiDB 版本：7.2.0
 
-试用链接：[快速体验](https://docs.pingcap.com/zh/tidb/v7.2/quick-start-with-tidb) | [下载离线包](https://cn.pingcap.com/product-community/?version=v7.2.0-DMR#version-list)
+试用链接：[快速体验](https://docs.pingcap.com/zh/tidb/v7.2/quick-start-with-tidb) | [下载离线包](https://cn.pingcap.com/product-community/)
```
Comment (Oreoxmt) [inline releases/release-7.2.0.md:12]:
```suggestion
试用链接：[快速体验](https://docs.pingcap.com/zh/tidb/v7.2/quick-start-with-tidb) | [下载离线包](https://cn.pingcap.com/product-community/?version=v7.2.0-DMR#version-list)
```

File: system-variables.md [hunk 2]
```diff
@@ -4491,7 +4473,7 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
 
 - 作用域：NONE
 - 默认值：`5.7.25-TiDB-(tidb version)`
-- 这个变量的值是 MySQL 的版本和 TiDB 的版本，例如 '5.7.25-TiDB-v7.3.0'。
+- 这个变量的值是 MySQL 的版本和 TiDB 的版本，例如 '5.7.25-TiDB-v7.2.0'。
```
Comment (Oreoxmt) [inline system-variables.md:4476]:
```suggestion
- 这个变量的值是 MySQL 的版本和 TiDB 的版本，例如 '5.7.25-TiDB-v7.3.0'。
```

File: tidb-resource-control.md [hunk 3]
```diff
@@ -81,12 +88,13 @@ Request Unit (RU) 是 TiDB 对 CPU、IO 等系统资源的统一抽象的计量
 
 ## 相关参数
 
-资源管控特性引入了两个新的全局开关变量：
+资源管控特性引入了如下系统变量或参数：
 
 - TiDB：通过配置全局变量 [`tidb_enable_resource_control`](/system-variables.md#tidb_enable_resource_control-从-v660-版本开始引入) 控制是否打开资源组流控。
 - TiKV：通过配置参数 [`resource-control.enabled`](/tikv-configuration-file.md#resource-control) 控制是否使用基于资源组配额的请求调度。
+- TiFlash：通过配置全局变量 [`tidb_enable_resource_control`](/system-variables.md#tidb_enable_resource_control-从-v660-版本开始引入) 和 TiFlash 参数 `enable_resource_control`（v7.4.0 开始引入）控制是否开启 TiFlash 资源管控。
```
Comment (Oreoxmt) [inline tidb-resource-control.md:95]: 缺少 TiFlash `enable_resource_control`/ `enable_pipeline` 的用户文档

File: releases/release-7.2.0.md (hunk 1 above)
Comment (joey-yez) [inline releases/release-7.2.0.md:12]: 这里为什么要修改 7.2 的 release note？

File: tidb-resource-control.md [hunk 4]
```diff
@@ -69,6 +74,8 @@ Request Unit (RU) 是 TiDB 对 CPU、IO 等系统资源的统一抽象的计量
     </tbody>
 </table>
 
+目前 TiFlash 资源管控仅考虑 SQL CPU 以及 read request payload。3 ms 消耗 1 RU，64 KiB read request payload 消耗 1 RU。
```
Comment (joey-yez) [inline tidb-resource-control.md:77]:
```suggestion
目前 TiFlash 资源管控仅考虑 SQL CPU 以及 read request payload。
```

Comment (joey-yez) [reply tidb-resource-control.md:77]: 建议这里不要提具体的 RU 对应的资源量。

File: tidb-resource-control.md [hunk 5]
```diff
@@ -343,6 +353,8 @@ Runaway Query 是指执行时间或消耗资源超出预期的查询。下面使
 
 2. 将 TiKV 参数 [`resource-control.enabled`](/tikv-configuration-file.md#resource-control) 设为 `false`，关闭按照资源组配额调度。
 
+3. 将 TiFlash 参数 `enable_pipeline` 设为 `false`，关闭 TiFlash 资源管控。
```
Comment (joey-yez) [inline tidb-resource-control.md:356]: 这里应该是调整 enable_resource_control 参数，不应该是调整 pipeline 参数。

File: system-variables.md (hunk 2 above)
Comment (guo-shaoge) [reply system-variables.md:4476]: Why 5.7.25-TiDB-v7.3.0 instead of 5.7.25-TiDB-v7.4.0 ?

File: tidb-resource-control.md (hunk 3 above)
Comment (guo-shaoge) [reply tidb-resource-control.md:95]: `enable_pipeline` is deprecated, so no need to add.
`enable_resource_control` is added

File: tidb-resource-control.md (hunk 5 above)
Comment (guo-shaoge) [reply tidb-resource-control.md:356]: done

File: system-variables.md (hunk 2 above)
Comment (guo-shaoge) [reply system-variables.md:4476]: done

File: tidb-resource-control.md [hunk 6]
```diff
@@ -12,6 +12,11 @@ TiDB 资源管控特性提供了两层资源管理能力，包括在 TiDB 层的
 - TiDB 流控：TiDB 流控使用[令牌桶算法](https://en.wikipedia.org/wiki/Token_bucket) 做流控。如果桶内令牌数不够，而且资源组没有指定 `BURSTABLE` 特性，属于该资源组的请求会等待令牌桶回填令牌并重试，重试可能会超时失败。
 - TiKV 调度：你可以为资源组设置绝对优先级 ([`PRIORITY`](/information-schema/information-schema-resource-groups.md#示例))，不同的资源按照 `PRIORITY` 的设置进行调度，`PRIORITY` 高的任务会被优先调度。如果没有设置绝对优先级 (`PRIORITY`)，TiKV 会将资源组的 `RU_PER_SEC` 取值映射成各自资源组读写请求的优先级，并基于各自的优先级在存储层使用优先级队列调度处理请求。
 
+从 v7.4.0 开始，TiDB 资源管控特性支持管控 TiFlash 资源，其原理与 TiDB 流控和 TiKV 调度类似：
+
+- TiFlash 流控：借助 [TiFlash Pipeline Model 执行模型](/tiflash/tiflash-pipeline-model.md)，可以更精确地获取不同查询的 CPU 消耗情况并转换为 RU 进行扣除，通过令牌桶算法进行流量控制
```
Comment (guo-shaoge) [inline tidb-resource-control.md:17]:
```suggestion
- TiFlash 流控：借助 [TiFlash Pipeline Model 执行模型](/tiflash/tiflash-pipeline-model.md)，可以更精确地获取不同查询的 CPU 消耗情况并转换为 RU 进行扣除，通过令牌桶算法进行流量控制。
```

File: tiflash/tiflash-configuration.md [hunk 7]
```diff
@@ -224,6 +224,9 @@ delta_index_cache_size = 0
     ## 从 v7.0.0 引入，表示带等值 join 条件的 HashJoin 算子在触发 spill 之前的最大可用内存，超过该阈值之后 HashJoin 算子会采用 spill to disk 的方式来减小内存使用。默认值为 0，表示内存使用无限制，即不会触发 spill。
     max_bytes_before_external_join = 0
 
+    ## 从 v7.4.0 引入，表示是否开启 TiFlash resource control 功能。
```
Comment (Oreoxmt) [inline tiflash/tiflash-configuration.md:227]:
```suggestion
    ## 从 v7.4.0 引入，表示是否开启 TiFlash 资源管控功能。当设置为 true 时，TiFlash 会使用 Pipeline Model 执行模型。
```

File: tidb-resource-control.md [hunk 8]
```diff
@@ -95,6 +103,8 @@ Request Unit (RU) 是 TiDB 对 CPU、IO 等系统资源的统一抽象的计量
 
 关于资源管控实现机制及相关参数的详细介绍，请参考 [RFC: Global Resource Control in TiDB](https://github.com/pingcap/tidb/blob/master/docs/design/2022-11-25-global-resource-control.md)。
 
+从 v7.4.0 开始，TiFlash 配置项 [`enable_resource_control`](/tiflash/tiflash-configuration.md#配置文件-tiflashtoml) 默认打开，与 [`tidb_enable_resource_control`](/system-variables.md#tidb_enable_resource_control-从-v660-版本开始引入) 一起配合，控制 TiFlash 资源管控的效果，只有二者都打开时，TiFlash 资源管控功能才会正常工作，否则 TiFlash 不会进行流控以及优先级调度。同时 TiFlash 配置项 [`enable_resource_control`](/tiflash/tiflash-configuration.md#配置文件-tiflashtoml) 打开时，会使用 [Pipeline Model 执行模型](/tiflash/tiflash-pipeline-model.md)。
```
Comment (Oreoxmt) [inline tidb-resource-control.md:106]:
```suggestion
从 v7.4.0 开始，TiFlash 配置项 `enable_resource_control` 默认打开，与 `tidb_enable_resource_control` 一起控制 TiFlash 资源管控功能。只有二者都启用时，TiFlash 资源管控功能才能进行流控以及优先级调度。同时，在开启 `enable_resource_control` 时，TiFlash 会使用 [Pipeline Model 执行模型](/tiflash/tiflash-pipeline-model.md)。

关于资源管控实现机制及相关参数的详细介绍，请参考 [RFC: Global Resource Control in TiDB](https://github.com/pingcap/tidb/blob/master/docs/design/2022-11-25-global-resource-control.md)。
```

File: tidb-resource-control.md [hunk 9]
```diff
@@ -12,6 +12,11 @@ TiDB 资源管控特性提供了两层资源管理能力，包括在 TiDB 层的
 - TiDB 流控：TiDB 流控使用[令牌桶算法](https://en.wikipedia.org/wiki/Token_bucket) 做流控。如果桶内令牌数不够，而且资源组没有指定 `BURSTABLE` 特性，属于该资源组的请求会等待令牌桶回填令牌并重试，重试可能会超时失败。
 - TiKV 调度：你可以为资源组设置绝对优先级 ([`PRIORITY`](/information-schema/information-schema-resource-groups.md#示例))，不同的资源按照 `PRIORITY` 的设置进行调度，`PRIORITY` 高的任务会被优先调度。如果没有设置绝对优先级 (`PRIORITY`)，TiKV 会将资源组的 `RU_PER_SEC` 取值映射成各自资源组读写请求的优先级，并基于各自的优先级在存储层使用优先级队列调度处理请求。
 
+从 v7.4.0 开始，TiDB 资源管控特性支持管控 TiFlash 资源，其原理与 TiDB 流控和 TiKV 调度类似：
+
+- TiFlash 流控：借助 [TiFlash Pipeline Model 执行模型](/tiflash/tiflash-pipeline-model.md)，可以更精确地获取不同查询的 CPU 消耗情况并转换为 RU 进行扣除，通过令牌桶算法进行流量控制。
+- TiFlash 调度：当系统资源不足时，多个资源组之间的 pipeline task 会根据优先级调度。具体逻辑包括：先判断资源组的优先级 `PRIORITY`，其次根据 CPU 使用情况，再结合 `RU_PER_SEC` 进行判断。最终效果：如果 rg1 和 rg2 的 `PRIORITY` 一样，但是 rg2 的 `RU_PER_SEC` 是 rg1 的两倍，那么 rg2 能使用的 CPU 时间就是 rg1 的两倍。
```
Comment (Oreoxmt) [inline tidb-resource-control.md:18]:
```suggestion
- TiFlash 流控：借助 [TiFlash Pipeline Model 执行模型](/tiflash/tiflash-pipeline-model.md)，可以更精确地获取不同查询的 CPU 消耗情况并转换为 [Request Unit (RU)](#什么是-request-unit-ru) 进行扣除。流量控制是通过令牌桶算法实现。
- TiFlash 调度：当系统资源不足时，多个资源组之间的 pipeline task 会根据优先级进行调度。具体逻辑是：首先判断资源组的优先级 `PRIORITY`，然后根据 CPU 使用情况，再结合 `RU_PER_SEC` 进行判断。最终效果是，如果 rg1 和 rg2 的 `PRIORITY` 一样，但是 rg2 的 `RU_PER_SEC` 是 rg1 的两倍，那么 rg2 可使用的 CPU 时间是 rg1 的两倍。
```

File: tidb-resource-control.md [hunk 10]
```diff
@@ -81,12 +88,13 @@ Request Unit (RU) 是 TiDB 对 CPU、IO 等系统资源的统一抽象的计量
 
 ## 相关参数
 
-资源管控特性引入了两个新的全局开关变量：
+资源管控特性引入了如下系统变量或参数：
 
 - TiDB：通过配置全局变量 [`tidb_enable_resource_control`](/system-variables.md#tidb_enable_resource_control-从-v660-版本开始引入) 控制是否打开资源组流控。
 - TiKV：通过配置参数 [`resource-control.enabled`](/tikv-configuration-file.md#resource-control) 控制是否使用基于资源组配额的请求调度。
+- TiFlash：通过配置全局变量 [`tidb_enable_resource_control`](/system-variables.md#tidb_enable_resource_control-从-v660-版本开始引入) 和 TiFlash 参数 [`enable_resource_control`](/tiflash/tiflash-configuration.md#配置文件-tiflashtoml)（v7.4.0 开始引入）控制是否开启 TiFlash 资源管控。
```
Comment (Oreoxmt) [inline tidb-resource-control.md:95]:
```suggestion
- TiFlash：通过配置全局变量 [`tidb_enable_resource_control`](/system-variables.md#tidb_enable_resource_control-从-v660-版本开始引入) 和 TiFlash 配置项 [`enable_resource_control`](/tiflash/tiflash-configuration.md#配置文件-tiflashtoml)（v7.4.0 开始引入）控制是否开启 TiFlash 资源管控。
```

File: tiflash/tiflash-pipeline-model.md [hunk 11]
```diff
@@ -7,71 +7,12 @@ summary: 介绍 TiFlash 新的执行模型 Pipeline Model。
...
-+------------------------------------+-------+
-| Variable_name                      | Value |
-+------------------------------------+-------+
-| tidb_enable_tiflash_pipeline_model | OFF   |
-+------------------------------------+-------+
-```
-
-变量 `tidb_enable_tiflash_pipeline_model` 支持 session 级别和 global 级别的修改。
+- v7.2.0 和 v7.3.0：TiFlash Pipeline Model 为实验特性，使用 [`tidb_enable_tiflash_pipeline_model`](https://docs.pingcap.com/zh/tidb/v7.2/system-variables#tidb_enable_tiflash_pipeline_model-从-v720-版本开始引入) 控制。
+- v7.4.0 及之后版本：TiFlash 正式支持 Pipeline Model。Pipeline Model 属于 TiFlash 内部特性，且 Pipeline Model 和 TiFlash resource control 功能相互依赖，因此，开启 TiFlash resource control 功能时，Pipeline Model 模型将自动启用。具体 TiFlash resource control 功能的启用方式和查看方式，请参考 [TiFlash Resource Control](/tidb-resource-control.md#相关参数)，同时变量 `tidb_enable_tiflash_pipeline_model` 被废弃。
```
Comment (Oreoxmt) [inline tiflash/tiflash-pipeline-model.md:13]:
```suggestion
- v7.4.0 及之后版本：Pipeline Model 成为正式功能。Pipeline Model 属于 TiFlash 内部特性，并且与 TiFlash 资源管控功能相互依赖，因此，开启 TiFlash 资源管控功能时，Pipeline Model 模型将自动启用。关于 TiFlash 资源管控功能的使用方式，参考[使用资源管控 (Resource Control) 实现资源隔离](/tidb-resource-control.md#相关参数)。同时，从 v7.4.0 开始，变量 `tidb_enable_tiflash_pipeline_model` 被废弃。
```

File: tiflash/tiflash-configuration.md (hunk 7 above)
Comment (joey-yez) [reply tiflash/tiflash-configuration.md:227]: 应该要强调 资源管控功能开启时，TiFlash 会使用 Pipeline Model 执行模型。

**Review by joey-yez — APPROVED** [review]

File: tidb-resource-control.md [hunk 12]
```diff
@@ -81,18 +88,21 @@ Request Unit (RU) 是 TiDB 对 CPU、IO 等系统资源的统一抽象的计量
...
+从 v7.0.0 开始，`tidb_enable_resource_control` 和 `resource-control.enabled` 开关都被默认打开。这两个参数的组合效果见下表：
 
 | `resource-control.enabled`  | `tidb_enable_resource_control`= ON | `tidb_enable_resource_control`= OFF  |
 |:----------------------------|:-----------------------------------|:------------------------------------|
 | `resource-control.enabled`= true  | 流控和调度（推荐组合）                        | 无效配置                         |
 | `resource-control.enabled`= false | 仅流控（不推荐）                           |  特性被关闭                   |
 
+从 v7.4.0 开始，TiFlash 配置项 `enable_resource_control` 默认打开，与 `tidb_enable_resource_control` 一起控制 TiFlash 资源管控功能。只有二者都启用时，TiFlash 资源管控功能才能进行流控以及优先级调度。同时，在开启 `enable_resource_control` 时，TiFlash 会使用 [Pipeline Model 执行模型](/tiflash/tiflash-pipeline-model.md)。
+
 关于资源管控实现机制及相关参数的详细介绍，请参考 [RFC: Global Resource Control in TiDB](https://github.com/pingcap/tidb/blob/master/docs/design/2022-11-25-global-resource-control.md)。
```
Comment (guo-shaoge) [inline tidb-resource-control.md:106]: todo 增加 tiflash resource control design dock link https://github.com/pingcap/tiflash/pull/8125

File: tiflash/tiflash-pipeline-model.md [hunk 13]
```diff
@@ -7,71 +7,12 @@ summary: 介绍 TiFlash 新的执行模型 Pipeline Model。
...
-+------------------------------------+-------+
-| Variable_name                      | Value |
-+------------------------------------+-------+
-| tidb_enable_tiflash_pipeline_model | OFF   |
-+------------------------------------+-------+
-```
-
-变量 `tidb_enable_tiflash_pipeline_model` 支持 session 级别和 global 级别的修改。
+- v7.2.0 和 v7.3.0：TiFlash Pipeline Model 为实验特性，使用 [`tidb_enable_tiflash_pipeline_model`](https://docs.pingcap.com/zh/tidb/v7.2/system-variables#tidb_enable_tiflash_pipeline_model-从-v720-版本开始引入) 控制。
+- v7.4.0 及之后版本：Pipeline Model 成为正式功能。Pipeline Model 属于 TiFlash 内部特性，并且与 TiFlash 资源管控功能相互依赖，因此，开启 TiFlash 资源管控功能时，Pipeline Model 模型将自动启用。关于 TiFlash 资源管控功能的使用方式，参考[使用资源管控 (Resource Control) 实现资源隔离](/tidb-resource-control.md#相关参数)。同时，从 v7.4.0 开始，变量 `tidb_enable_tiflash_pipeline_model` 被废弃。
```
Comment (guo-shaoge) [inline tiflash/tiflash-pipeline-model.md:13]:
```suggestion
- v7.4.0 及之后版本：Pipeline Model 成为正式功能。Pipeline Model 属于 TiFlash 内部特性，并且与 TiFlash 资源管控功能绑定，开启 TiFlash 资源管控功能时，Pipeline Model 模型将自动启用。关于 TiFlash 资源管控功能的使用方式，参考[使用资源管控 (Resource Control) 实现资源隔离](/tidb-resource-control.md#相关参数)。同时，从 v7.4.0 开始，变量 `tidb_enable_tiflash_pipeline_model` 被废弃。
```

File: tidb-resource-control.md (hunk 12 above)
Comment (guo-shaoge) [reply tidb-resource-control.md]: done

File: tidb-resource-control.md [hunk 14]
```diff
@@ -81,19 +88,22 @@ Request Unit (RU) 是 TiDB 对 CPU、IO 等系统资源的统一抽象的计量
...
 
 | `resource-control.enabled`  | `tidb_enable_resource_control`= ON | `tidb_enable_resource_control`= OFF  |
 |:----------------------------|:-----------------------------------|:------------------------------------|
 | `resource-control.enabled`= true  | 流控和调度（推荐组合）                        | 无效配置                         |
 | `resource-control.enabled`= false | 仅流控（不推荐）                           |  特性被关闭                   |
 
-关于资源管控实现机制及相关参数的详细介绍，请参考 [RFC: Global Resource Control in TiDB](https://github.com/pingcap/tidb/blob/master/docs/design/2022-11-25-global-resource-control.md)。
+从 v7.4.0 开始，TiFlash 配置项 `enable_resource_control` 默认打开，与 `tidb_enable_resource_control` 一起控制 TiFlash 资源管控功能。只有二者都启用时，TiFlash 资源管控功能才能进行流控以及优先级调度。同时，在开启 `enable_resource_control` 时，TiFlash 会使用 [Pipeline Model 执行模型](/tiflash/tiflash-pipeline-model.md)。
+
+关于资源管控实现机制及相关参数的详细介绍，请参考 [RFC: Global Resource Control in TiDB](https://github.com/pingcap/tidb/blob/master/docs/design/2022-11-25-global-resource-control.md) 以及 [TiFlash Resource Control Design Doc](https://github.com/pingcap/tiflash/blob/master/docs/design/2023-09-21-tiflash-resource-control.md)
```
Comment (Oreoxmt) [inline tidb-resource-control.md:106]:
```suggestion
关于资源管控实现机制及相关参数的详细介绍，请参考 [RFC: Global Resource Control in TiDB](https://github.com/pingcap/tidb/blob/master/docs/design/2022-11-25-global-resource-control.md) 以及 [TiFlash Resource Control](https://github.com/pingcap/tiflash/blob/master/docs/design/2023-09-21-tiflash-resource-control.md)。
```

**Review by Oreoxmt — APPROVED** [review]

**Review by SeaRise — APPROVED** [review]

File: tidb-resource-control.md [hunk 15]
```diff
@@ -69,6 +74,8 @@ Request Unit (RU) 是 TiDB 对 CPU、IO 等系统资源的统一抽象的计量
     </tbody>
 </table>
 
+目前 TiFlash 资源管控仅考虑 SQL CPU 以及 read request payload。
```
[the remark this answers is not in the document]
Comment (guo-shaoge) [reply tidb-resource-control.md:77]: "目前 TiFlash 资源管控仅考虑 SQL CPU (即查询的 pipeline task 运行所占用的 CPU 时间) 以及 read request payload。"

[the remark this answers is not in the document]
Comment (guo-shaoge) [reply tidb-resource-control.md:77]: @ran-huang 这样 OK 不

Comment (guo-shaoge) [inline tidb-resource-control.md:77]:
```suggestion
目前 TiFlash 资源管控仅考虑 SQL CPU (即查询的 pipeline task 运行所占用的 CPU 时间) 以及 read request payload。
```

File: tidb-resource-control.md [hunk 16]
```diff
@@ -69,6 +74,8 @@ Request Unit (RU) 是 TiDB 对 CPU、IO 等系统资源的统一抽象的计量
     </tbody>
 </table>
 
+目前 TiFlash 资源管控仅考虑 SQL CPU (即查询的 pipeline task 运行所占用的 CPU 时间) 以及 read request payload。
```
Comment (Oreoxmt) [inline tidb-resource-control.md:77]:
```suggestion
目前 TiFlash 资源管控仅考虑 SQL CPU（即查询的 pipeline task 运行所占用的 CPU 时间）以及 read request payload。
```

**Review by ran-huang — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

**Pull request merged**